README.md

```diff
@@ -3,6 +3,7 @@
 
 ## Project setup
 ```
+cp .lifelinesrc.example .lifelinesrc
```

Comment:
The Jenkinsfile tries to build the artifact for CI. So add this to the Jenkinsfile too I think

Reply:
Added the copy action to both stages of Jenkins